.circleci/config.yml

```
@@ -128,11 +128,11 @@ jobs:
       - run: *check_skip_job
       - restore_cache:
           keys:
-            - build-v5-{{ .Branch }}-{{ .Revision }}
-            - build-v5--{{ .Revision }}
-            - build-v5-{{ .Branch }}-
-            - build-v5-master-
-            - build-v5-
+            - build-v6-{{ .Branch }}-{{ .Revision }}
```

new year, fresh build cache?
```suggestion
            - build-v0-{{ .Branch }}-{{ .Revision }}
```

---

Now that we have a build cache, I'd rather not waste extra cycles rebuilding from scratch. Happy to add it to our year-end todo list...